.cm/change_categories.cm

```diff
@@ -192,6 +192,7 @@ platforms:
     name: 'ide'
     subprojects:
       - 'subprojects/ide/'
+      - 'platforms/ide/ide-plugins/'
```

Comment:
It doesn't necessarily need to be done in this PR, but IMO one of the intentions of platformization is that we shouldn't need to list individual subprojects here. For example in codeowners we only need to list `platforms/ide`, whereas here we need to list `platforms/ide/ide-plugins`. IMO we should rename the `subprojects:` field to something like `directories` or `subdirs` so we don't feel like we need to explicitly list all subprojects here. Altnernatively, we could omit the `platform/ide` directory altogether by assuming there is a directory in `platforms/` matching the platform name in this file

Reply:
This is a good idea, I'll make a note to add this to my next gitStream update.